roles/aks-deployment/files/digitransit-deployer-dev.yml

```diff
@@ -0,0 +1,100 @@
+---  
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: digitransit-deployer
+  namespace: default
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: digitransit-deployer-role
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+    verbs:
+      - list
+  - apiGroups:
+      - "apps"
+    resources:
+      - deployments
+    verbs:
+      - list
+      - patch
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: digitransit-deployer-rolebind
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: digitransit-deployer-role
+subjects:
+- kind: ServiceAccount
+  name: digitransit-deployer
+  namespace: default
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: digitransit-deployer
+spec:
+  ports:
+    - port: 10026
+      name: digitransit-deployer-service-port
+      protocol: TCP
+  selector:
+    app: digitransit-deployer
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: digitransit-deployer
+  labels:
+    app: digitransit-deployer
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 25%
+  selector:
+    matchLabels:
+      app: digitransit-deployer
+  template:
+    metadata:
+      labels:
+        app: digitransit-deployer
+        update: auto
+    spec:
+      serviceAccountName: digitransit-deployer
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                - key: app
+                  operator: Exists
+              topologyKey: kubernetes.io/hostname
+      containers:
+      - name: digitransit-deployer
+        image: hsldevcom/digitransit-deployer:kubernetes
+        imagePullPolicy: Always
+        securityContext:
+          allowPrivilegeEscalation: false
+        env:
+        - name: DEBUG
+          value: "*"
+        resources:
+          requests:
+            memory: 256Mi
+          limits:
+            memory: 256Mi
+            cpu: 1
```

Comment:
CPU values are used like this: 0.1cpu is in kubernetes deployment 100m. If this means 1 cpu this should be 1000m.

Reply:
Fixed